components/infobox/extensions/commons/age_calculation.lua

```diff
@@ -0,0 +1,154 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:AgeCalculation
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local AgeCalculation = {}
+local Class = require('Module:Class')
+
+local _LANG = mw.getContentLanguage()
+
+local _EPOCH = _LANG:formatDate('c', '1970-1-1')
+local _EPOCH_FIELD = { 1970, 1, 1 }
+local _CURRENT_FIELD = os.date('*t', os.time())
+local _CURRENT_ISO = _LANG:formatDate('c')
+local _CURRENT_YEAR = tonumber(_LANG:formatDate('Y'))
+local _DAYS_IN_MONTH = { 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 }
```

Comment:
Leap years?

Reply:
renamed the var
it is only used to determine a fallback value, that fallback value then is checked against current date (in a way that it is irrelevant if the 29th of februaray exists) and used in the age calculation function, but i account for that case there